### PR TITLE
[9.0][FIX] purchase_request_to_rfq: Fix store attribute and search filter view in purchase.request.line model

### PR DESCRIPTION
**Pull request by sergio-teruel**
Minor fixes.
cc @Tecnativa @jbeficent 

Thanks

### CODE REVIEW

File: purchase_request_to_rfq/views/purchase_request_view.xml [hunk 1]
```diff
@@ -75,7 +75,7 @@
                     <field name="date_required" position="after">
                         <filter name="purchase_state_none"
                                 string="Purchasing not started"
-                                domain="[('purchase_state','=','none')]"
+                                domain="[('purchase_state','=', False)]"
```
Comment (pedrobaeza) [inline purchase_request_to_rfq/views/purchase_request_view.xml:78]: Are you sure about this change?

Comment (JordiBForgeFlow) [reply purchase_request_to_rfq/views/purchase_request_view.xml:78]: His proposed change is good.

**Review by pedrobaeza — APPROVED** [review]

**Review by JordiBForgeFlow — APPROVED** [review]
:+1: Technical and functional review

**Comment by JordiBForgeFlow** [comment]
@aheficent Can you review?

**Comment by JordiBForgeFlow** [comment]
cc @adrienpeiffer

**Review by adrienpeiffer — APPROVED** [review]
:+1:

**Comment by sergio-teruel** [comment]
Thank... super fast :smile:

**Review by AaronHForgeFlow — COMMENTED** [review]
:+1:

**Pull request merged**